### ISSUE TITLE
Resolution options that match some game's internal resolutions?

### ISSUE THREAD

**Issue by MessiahMozgus** [xerpi/vita-udcd-uvc#20]
Just curious if it would be possible for the plugin to be set to some resolutions like 720x408, 704x448, 640x384, and others mentioned over here: https://github.com/Electry/VitaGrafixPatchlist#supported-games

This would allow for better image quality I would imagine since you would be upscaling on your PC only once, rather than twice, assuming you opted for 864x488 to maintain 60fps.

Of course we could also use the VitaGraphics plugin to modify the game resolutions ourselves to match 864x488, I assume, but I have run into issues with some games not mentioned on that compatibility list...

**Comment by xerpi** [comment]
Hi, this could be possible indeed, you take a look at the currently supported resolutions here (it very easy to add a new entry): https://github.com/xerpi/vita-udcd-uvc/blob/master/usb_descriptors.h#L132

The problem is that the resolution is chosen at PC side, so the user would have to know the game's original resolution and select it,

**Comment by MessiahMozgus** [comment]
I'd be very happy if you could. We have that table to tell us the resolutions for lots of games, plus there are plugins that can tell us as much, aren't there? And when we take screenshots, do they snap at game res or Vita max res? I never checked. Might be another way to tell.

**Comment by xerpi** [comment]
That's right, but it is the PC side (OBS, KMPlayer, etc) who selects which resolution to use. This means that if the user selects 960x544 on the PC side, even if the game the PSVita is running renders at 704x448, it will upscale it.

**Comment by MessiahMozgus** [comment]
Understood. I'm really not seeing a downside. People who do their homework will get a better experience per title. people who dont will be in the same experience they are currently getting.
